var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -126,13 +126,20 @@ class Boost(Package):
     depends_on('mpi', when='+mpi')
     depends_on('bzip2', when='+iostreams')
     depends_on('zlib', when='+iostreams')
+    depends_on('zlib', when='%xl_r' )
+    depends_on('bzip2', when='%xl_r')
 
     # Patch fix from https://svn.boost.org/trac/boost/ticket/11856
     patch('boost_11856.patch', when='@1.60.0%gcc@4.4.7')
 
     # Patch fix from https://svn.boost.org/trac/boost/ticket/10125
     patch('boost_10125.patch', when='@1.55.0%gcc@5.0:5.9')
 
+    # Patch fix for IBM XL compiler
+    patch('xl_1_62_0_le.patch', when='@1.62.0%xl_r')
```

Comment:
Is this the only version we can build with XL? Or does the patch apply just as well to versions other than 1.62.0? And what about regular `xl`?